src/areas/home/home.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react';
+import classnames from 'classnames/bind';
+import { useTranslation } from '../../i18n';
+const cx = classnames.bind(require('./home.module.scss'));
+
+export default function Home() {
+    const [loc] = useTranslation();
+    return (
+        <>

```

Comment:
nit: should we still have <> when there is one child element

Reply:
good catch, removed.